tests/integration/shared/sources/gitdiff.config.example

```diff
@@ -0,0 +1,4 @@
+{
+  "baseRemoteName": "upstream",
+  "baseBranchName": "master"
+}
```

Comment:
Can we include some JSON "comments" about what the user is expected to do in this file?

Reply:
Added, thanks, great idea.